src/main/java/uk/gov/ch/repository/update/OverseasEntityManagingOfficersRepository.java

```diff
@@ -12,24 +12,24 @@ public interface OverseasEntityManagingOfficersRepository extends PagingAndSorti
     static final String OVERSEAS_ENTITY_MANAGING_OFFICERS_TYPE_ID = "5013";
```

Comment:
Is there test for the repository interfaces?

Reply:
If not can be done as tech debt.

Reply:
No tests are added for the repository interfaces as part of our implementation. The calling of the repository methods are tested in the associated service classes and functionality of the query is manually tested once merged in